--- a/internal/demo/helloworld/worker.go
+++ b/internal/demo/helloworld/worker.go
@@ -29,8 +29,12 @@
 		return "", err
 	}
 
+	if err := workflow.Sleep(ctx, 5*time.Second); err != nil {
+		return "", err
+	}
+
 	// Return the greeting
-	return fmt.Sprintf("Hello %s (time-gated v2)", subject), nil
+	return fmt.Sprintf("Hello %s (time-gated v3)", subject), nil
 }
 
 func GetSubject(ctx context.Context) (string, error) {
